cmd/plasmacli/subcmd/eth/query/root.go

```diff
@@ -16,6 +16,7 @@ var (
 	indexF    = "index"
 	limitF    = "limit"
 	positionF = "position"
+	useNodeF  = "use-node"
```

Comment:
I think you should be able to use the "trust node" flag. Should be imported from cosmos flags

Reply:
+1